File: src/main/java/br/com/leonardoferreira/jirareport/domain/embedded/ColumnTimeAvg.java
package br.com.leonardoferreira.jirareport.domain.embedded;

import java.beans.Transient;
import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ColumnTimeAvg implements Serializable {
    private static final long serialVersionUID = -3791819163293059573L;

    private String columnName;

    private Double avgTime;

    @Transient
    public String getLeadTime() {
        return String.format("%.2f", avgTime);
    }
}

File: src/main/kotlin/br/com/jiratorio/domain/entity/embedded/ColumnTimeAvg.kt
package br.com.jiratorio.domain.entity.embedded

import java.io.Serializable

data class ColumnTimeAvg(

    val columnName: String,

    val avgTime: Double

) : Serializable {
    companion object {
        private const val serialVersionUID = -3791819163293059573L
    }
}
